src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs

```diff
@@ -13,74 +14,79 @@
 
 namespace Lykke.Job.TransactionHandler.Queues
 {
-    public sealed class TransferQueue : IQueueSubscriber
+    public sealed class TransferQueue : IStartable, IDisposable
     {
         private const bool QueueDurable = true;
 
         private readonly ILog _log;
+        private readonly string _mongoConnectionString;
```

Comment:
I would prefer to use RabbitMqSettings and MongoDeduplicator instead of lot of strings.